app/views/admin/reports/_rendering_options.html.haml

```diff
@@ -31,12 +25,7 @@
 
 - if @report.available_headers.present?
   .row
-    - if feature? :report_inverse_columns_logic, spree_current_user
-      .alpha.two.columns= label_tag nil, t(:report_columns)
-      .omega.fourteen.columns
-        = render MultipleCheckedSelectComponent.new(name: "fields_to_show", options: @report.available_headers, selected: @params_fields_to_show)
-    - else
-      .alpha.two.columns= label_tag nil, t(:report_hide_columns)
-      .omega.fourteen.columns
-        = select_tag(:fields_to_hide, options_for_select(@report.available_headers, params[:fields_to_hide]),
-                    class: "select2 fullwidth", multiple: true)
+    .alpha.two.columns= label_tag nil, t(:report_columns)
+    .omega.fourteen.columns
+      = render MultipleCheckedSelectComponent.new(name: "fields_to_show", options: @report.available_headers, selected: @params_fields_to_show)
+  
```

Comment:
:scream: :wink:

Reply:
Not sure to understand ... ! ;)

Reply:
Not visible here but in the diff:
![Screenshot from 2022-10-25 11-05-37](https://user-images.githubusercontent.com/3524483/197652785-6684f34f-4fc1-49fe-820f-716847ea3a3b.png)

You added two spaces into an otherwise empty line and then finished the file without a newline. I'm surprised that Rubocop didn't shoot you. :wink: